lib/unison-pretty-printer/package.yaml

```diff
@@ -0,0 +1,76 @@
+name: unison-pretty-printer
+github: unisonweb/unison
+copyright: Copyright (C) 2013-2022 Unison Computing, PBC and contributors
+
+default-extensions:
+  - ApplicativeDo
+  - BangPatterns
+  - BlockArguments
+  - DeriveFunctor
+  - DeriveGeneric
+  - DerivingStrategies
+  - DoAndIfThenElse
+  - FlexibleContexts
+  - FlexibleInstances
+  - GeneralizedNewtypeDeriving
+  - LambdaCase
+  - MultiParamTypeClasses
+  - NamedFieldPuns
+  - OverloadedStrings
+  - PatternSynonyms
+  - RankNTypes
+  - ScopedTypeVariables
+  - TupleSections
+  - TypeApplications
+  - ViewPatterns
+
+ghc-options: -Wall -O0 -fno-warn-name-shadowing -fno-warn-missing-pattern-synonym-signatures
+
+flags:
+  optimized:
+    manual: true
+    default: true
+
+when:
+  - condition: flag(optimized)
```

Comment:
Need to update `release.yaml` to pass this flag too; or maybe make the default be `-O2` and just use `--fast` to get `-O0`.

Reply:
We currently set the flag on by default in all cases I think, see the lines above.

```
flags:
  optimized:
    manual: true
    default: true
```